debian10/autogenerated_dus_timestamp.txt

```diff
@@ -0,0 +1 @@
+Mon Sep 30 13:03:27 UTC 2019
```

Comment:
You actually don't have to create this file manually.
It will be created by the first DUS run.